definitions/npm/aphrodite_v2.x.x/flow_v0.25.x-/aphrodite_v2.x.x.js

```diff
@@ -0,0 +1,401 @@
+  tabSize?: string;
+  textAlign?: string;
+  textAlignLast?: string;
+  textCombineUpright?: string;
+  textDecoration?: string;
+  textDecorationColor?: string;
+  textDecorationLine?: string;
+  textDecorationStyle?: string;
+  textIndent?: string;
+  textOrientation?: string;
+  textOverflow?: string;
+  textRendering?: string;
+  textShadow?: string;
+  textTransform?: string;
+  textUnderlinePosition?: string;
+  top?: string | number;
+  touchAction?: string;
+  transform?: string;
+  transformOrigin?: string;
+  transformStyle?: string;
+  transition?: string;
+  transitionDelay?: string;
+  transitionDuration?: string;
+  transitionProperty?: string;
+  transitionTimingFunction?: string;
+  turn?: string;
+  unicodeBidi?: string;
+  unicodeRange?: string;
+  userSelect?: string;
+  verticalAlign?: string;
+  visibility?: string;
+  webkitOverflowScrolling?: string;
+  webkitTransform?: string;
+  webkitTransformOrigin?: string;
+  webkitTransitionDelay?: string;
+  webkitTransitionDuration?: string;
+  webkitTransitionProperty?: string;
+  webkitTransitionTimingFunction?: string;
+  whiteSpace?: string;
+  widows?: string;
+  width?: string | number;
+  willChange?: string;
+  wordBreak?: string;
+  wordSpacing?: string;
+  wordWrap?: string;
+  writingMode?: string;
+  zIndex?: string | number;
+  [key: string]: any,
+  ...
+};
+
+type SelectorCallback = (selector: string) => Array<string>;
+
+type SelectorHandler = (
+    selector: string,
+    baseSelector: string,
+    callback: SelectorCallback
+) => Array<string> | string | null;
+
+type SheetDefinition = {|
+  _definition: StyleObject,
+  _len: number,
+  _name: string,
+|};
+
+type MaybeSheetDefinition = SheetDefinition | false | null | void;
+
+type InjectAndGetClassName = (
+  useImportant: boolean,
+  styleDefinitions: Array<MaybeSheetDefinition>,
+  selectorHandlers: Array<SelectorHandler>,
+) => string;
+
+type StyleSheetServer = {|
+  renderStatic: (renderFunc: () => string) => {|
+    html: string,
+    css: {|
+      content: string,
+      renderedClassNames: Array<string>,
+    |},
+  |}
+|};
+
+type StyleSheetTestUtils = {|
+  suppressStyleInjection: () => void,
+  clearBufferAndResumeStyleInjection: () => void,
+  getBufferedStyles: () => Array<string>,
+|};
+
+type Aphrodite = {
+  StyleSheet: {|
+    create: (
+      sheetDefinition: {
+        [key: string]: StyleObject,
+        ...
+      },
+    ) => {
+      [key: string]: SheetDefinition,
+      ...
+    },
+    rehydrate: (renderedClassNames?: Array<string>) => void,
+  |},
+  StyleSheetServer: StyleSheetServer,
+  StyleSheetTestUtils: StyleSheetTestUtils,
+  minify: (shouldMinify: boolean) => void,
+  css: (...styleDefinitions: Array<MaybeSheetDefinition>) => string,
+  flushToStyleTag: () => void,
+  injectAndGetClassName: InjectAndGetClassName,
+  defaultSelectorHandlers: Array<SelectorHandler>,
+  reset: () => void,
+  resetInjectedStyle: (key: string) => void,
+  ...
+};
+
+declare module 'aphrodite' {
```

Comment:
I don't think they're globally usable because I haven't used the `declare` syntax. But also I need to define them outside of the module because you can import them from both, `aphrodite` or `aphrodite/no-important`

Reply:
Actually are you saying that all these types would conflict with types outside of this file? So I should name them as `Aphrodite$StyleObject`?

Reply:
Ok done

Reply:
Since both modules are defined **in the same file**, they could share (import) types between each others.